torch/onnx/symbolic_opset9.py

```diff
@@ -1591,6 +1591,30 @@ def pow(g, self, exponent):
     return pow
 
 
+def _clamp_common_types(g, self, min_f=None, max_f=None):
+    assert not (
+        min is None and max is None), "either max or min should be specified"
+
+    # clip-6 is only available to float, double, and float16.
+    # for other types, we first cast to float, then do clip and cast back.
+    origin_dtype = self.type().scalarType()
+    is_fp_clamp = self.type().scalarType(
+    ) == "Float" or self.type().scalarType() == "Double"
+
+    if not is_fp_clamp:
+        warnings.warn(
+            "ONNX opset < 11 does not support clamp/clip to non-float types. Using cast-clip-cast instead.")
```

Comment:
It does not make sense reference a future opset in this message, right?

Reply:
If this implementation belongs to opset 6, move this piece of code there.

Reply:
On a second note, there is no `symbolic_opset6.py` in the repo. Maybe we should leave it on symbolic 9 and check with pytorch folks what is going on.